MyLibrary/Tests/PodTests.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  PodTests.swift
+//  MyLibrary
+//
+//  Created by Rafael Ramos on 17/05/23.
+//
+
+import XCTest
+
+final class PodTests: XCTestCase {
+
+    override func setUpWithError() throws {
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+    }
+
+    override func tearDownWithError() throws {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+    }
+
+    func test_pods() throws {
+        XCTAssertEqual(1, 9)
```

Comment:
<!--
  1 failure:  **MyLibrary_Unit_...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: MyLibrary/Tests/PodTests.swift;
  Line: 21;
-->
- :no_entry_sign: **MyLibrary_Unit_Tests.PodTests: test_pods**<br />XCTAssertEqual failed: ("1") is not equal to ("9")